### DISCUSSION TITLE
Using Uno platform without Uno platform styles

### DISCUSSION THREAD

**Discussion by ArchontBelketh** [unoplatform/uno#10911] [Q&A] [answered]
Hi!
I just started learning Figma, Uno platform and Visual Studio 2022.
In VS 2022 i create Uno Platform App project.
Can i use Uno platform without using Uno platform styles?
If so, where can I read about it?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for trying out Uno Platform! Can you explain what you mean by "Uno Platform styles" ?

**Reply by ArchontBelketh** [reply]
I create some simple form in Blend for VS.
![example1](https://user-images.githubusercontent.com/122164798/211607459-08b8a240-55b6-40a8-af53-c70cd8f501c8.png)
When I copy XAML code to project and run .wasm application form becomes like this
![example](https://user-images.githubusercontent.com/122164798/211608612-b3141419-8531-454c-a8dd-a6489d402afd.png)
Looks like some kind of styling has been applied to the form elements.
I saw this style in Figma when I imported the Uno platform material toolkit. Therefore, I considered that this style of Uno platform.
Is it possible to ignore the application of styling so that the form is displayed as in the first screenshot?

**Reply by jeromelaban** [reply]
Thanks. What you're seeing is the Material style indeed, but you can disable material altogether as needed in the `App.xaml` file. What does your XAML for this UI look like?

**Reply by ArchontBelketh** [reply]
Thanks for your answer!
![Appxaml](https://user-images.githubusercontent.com/122164798/211617001-dbb83c87-68e6-41e7-bf70-5b5e9e13cda0.png)
![mainpagexaml](https://user-images.githubusercontent.com/122164798/211632386-dac4d73e-2c19-4d06-8de5-86f400ffdb97.png)

**Reply by jeromelaban** [reply]
Thanks! You can remove `<Material ...`  entries from the `app.xaml` file, it will give you the Microsoft Fluent styles.

**Reply by ArchontBelketh** [reply]
It works! Thank you for answer :)
